src/centrifuge.js

```diff
@@ -735,11 +755,17 @@ export class Centrifuge extends EventEmitter {
           if (resolveCtx.next) {
             resolveCtx.next();
           }
+          if (this._config.onRefreshAttempt) {
+            this._config.onRefreshAttempt(true, resolveCtx);
+          }
         }, rejectCtx => {
           this._refreshError(rejectCtx.error);
           if (rejectCtx.next) {
             rejectCtx.next();
           }
+          if (this._config.onRefreshAttempt) {
+            this._config.onRefreshAttempt(false, rejectCtx);
```

Comment:
the same note about context here - we need to create new object with required exported fields only.

Reply:
Updated, see if this is what you meant